### PR TITLE
List ADD

### PR DESCRIPTION
**Pull request by VadimDrobyazko**
[DEMO LINK](https://vadimdrobyazko.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,45 +1,88 @@
...
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, SetCount] = useState(0);
```
Comment (mykhalenych) [inline src/components/NewMovie/NewMovie.tsx:11]: why all setters starts with capital letter?
use camelCase

**Review by mykhalenych — CHANGES_REQUESTED** [review]
add validation to inputs
Screenshot 2023-04-03 at 17 12 20

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Screenshot 2023-04-04 at 17 34 08
how can the user understand what he entered incorrectly?

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,29 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
+import { Movie } from './types/Movie';
+
 export const App = () => {
+  const [movies, SetNewMovies] = useState(moviesFromServer);
```
Comment (mykhalenych) [inline src/App.tsx:10]: change this setter as well

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**